python/test/compile_api/tracing.py

```diff
@@ -59,14 +59,14 @@ def forward(self, x):
 
 
 try:
-    # CHECK: Only Tensors, TensorPlaceholders, or a sequences of Tensors and TensorPlaceholders are supported as inputs.
+    # CHECK: Only Tensors, TensorPlaceholder's, or sequences of Tensors and TensorPlaceholder's are supported as example args for method inputs. Got '{'a': tensor(3.)}'
```

Comment:
Nit: an apostrophe here implies it's possessive, not plural

Reply:
I'm in the camp where making sure that the identifier boundaries are correct and greppable takes precedence over the grammatical rules of ordinary English prose. In Markdown I would typically write `TensorPlaceholder`s, but this is a plaintext message.

Reply:
Makes sense. Should Tensors be changed to "Tensor's" too then?

Reply:
Oh duh, yes lol

Reply:
https://github.com/llvm/torch-mlir/pull/1569